src/main/java/core/basesyntax/service/operations/OperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface OperationHandler {
+    void getOperation(List<FruitTransaction> transactions);

```

Comment:
The method signature for handling operations should not directly accept a list of `FruitTransaction`. Instead, it should accept a single `FruitTransaction` and perform the operation on it. Handling multiple transactions should be done at a higher level, such as in a service class that iterates over transactions and calls the appropriate `OperationHandler` for each one.